guides/user/managed-delivery/delivery-configs/index.md

```diff
@@ -130,26 +130,21 @@ environments:
   notifications: # omitted for brevity
   constraints: []
   resources: # details omitted for brevity
-  - apiVersion: bakery.spinnaker.netflix.com/v1
-    kind: image
+  - kind: bakery/image@v1
```

Comment:
We can even remove this entirely. Image resources are no longer a thing

Reply:
+1 but I'd additionally suggest adding a little hint/redacted `vmOptions` field on the deb artifact for completeness, e.g.:

```yaml
artifacts:
- name: keeldemo 
  type: deb
  reference: my-artifact
  vmOptions:
    # omitted for brevity
```

Reply:
THis as done in #1801